Sources/MapboxNavigation/UserHaloCourseView.swift

```diff
@@ -5,28 +5,42 @@ import CoreLocation
  A view representing the user’s reduced accuracy location on screen.
  */
 open class UserHaloCourseView: UIView, CourseUpdatable {
-    private var lastLocationUpdate: Date?
 
-    // Sets the inner fill color of the user halo
+    /**
+     Sets the inner fill color of the user halo.
+     */
     @objc public dynamic var haloColor: UIColor = #colorLiteral(red: 1, green: 1, blue: 1, alpha: 0.5) {
         didSet {
             haloView.haloColor = haloColor
         }
     }
 
-    // Sets the ring fill color of the circle around the user halo
+    /**
+     Sets the ring fill color of the circle around the user halo.
+     */
     @objc public dynamic var haloRingColor: UIColor = #colorLiteral(red: 0.149, green: 0.239, blue: 0.341, alpha: 0.3) {
         didSet {
             haloView.haloRingColor = haloRingColor
         }
     }
 
-    // Sets the ring size by the radius of the user halo
+    /**
+     Sets the ring size by the radius of the user halo.
+     */
     @objc public dynamic var haloRadius: Double = 100.0 {
         didSet {
             haloView.haloRadius = haloRadius
         }
     }
+    
+    /**
+     Sets the halo ring border width.
+     */
+    @objc public dynamic var haloBorderWidth: Double = 5.0 {
```

Comment:
Let’s mention the addition of this property in the changelog, even if it a bit of a passthrough.